src/Sinch/Verification/Start/Request/StartCalloutVerificationRequest.cs

```diff
@@ -10,5 +10,10 @@ public class StartCalloutVerificationRequest : StartVerificationRequestBase
         /// </summary>
         [JsonInclude]
         public override VerificationMethodEx Method { get; } = VerificationMethodEx.Callout;
+
+        /// <summary>
+        ///     Text-To-Speech engine setting. A language-region identifier according to IANA. Only a subset of those identifiers is accepted.
+        /// </summary>
+        public string? Locale { get; set; }
```

Comment:
This doesn't seem to be related to the PR scope.
Keep in mind that this property is part of an object. A similar object is defined in the callback (with a different description) and states that the upper element "speech" can define various properties.